doc/source/cookbook/simple_projection_methods.py

```diff
@@ -0,0 +1,13 @@
+import yt
+
+# Load the dataset.
+ds = yt.load("GalaxyClusterMerger/fiducial_1to3_b0.273d_hdf5_plt_cnt_0175")
+
+# Create projections of temperature (with different methods)
+
+
+for method in ["integrate", "min", "max"]:
+    proj = yt.ProjectionPlot(ds, "x", ("gas", "temperature"), method=method)
+    proj.save()
+## note, the default save name doesn't look at the method
+## so otherwise it'd just overwrite each loop through
```

Comment:
I think you forgot to keep your custom file name in refactoring. Here's a suggestion to fix it (I'm also removing the comment because this doesn't need to be justified in the recipe)

```suggestion
    proj.save(f"projection_method_{method}.png")
```